gui/wxpython/datacatalog/catalog.py

```diff
@@ -51,16 +62,26 @@ def __init__(self, parent, giface=None, id=wx.ID_ANY,
 
     def _layout(self):
         """Do layout"""
-        sizer = wx.BoxSizer(wx.VERTICAL)
+        self.sizer = wx.BoxSizer(wx.VERTICAL)
+        self.sizer.Add(self.toolbar, proportion=0, flag=wx.EXPAND)
+        self.sizer.Add(self.tree.GetControl(), proportion=1, flag=wx.EXPAND)
+
+        # infobar instance
+        self.infoBarDataCatalog = InfoBar(self)
 
-        sizer.Add(self.toolbar, proportion=0,
-                  flag=wx.EXPAND)
+        # instance of infobar manager for data catalog
+        self.infoManagerDataCatalog = InfoManagerDataCatalog(infobar=self.infoBarDataCatalog,
+                                                             sizer=self.sizer)
 
-        sizer.Add(self.tree.GetControl(), proportion=1,
-                  flag=wx.EXPAND)
+        # Show first infobar for first-time user
+        if is_current_mapset_in_demolocation:
```

Comment:
The function is not called, but the expression is always true. Test this code in something which is not a demo location.